### PR TITLE
Add 4326 tile rendering support

### PR DESCRIPTION
**Pull request by meilinger**

### CODE REVIEW

**Comment by meilinger** [comment]
4326 OL5 test harness: [index.html.zip](https://github.com/go-spatial/tegola/files/3698886/index.html.zip)

Example config using OSM data: [tegola.toml.zip](https://github.com/go-spatial/tegola/files/3698895/tegola.toml.zip)

File: atlas/map.go [hunk 1]
```diff
@@ -207,11 +204,12 @@ func (m Map) Encode(ctx context.Context, tile *slippy.Tile) ([]byte, error) {
 					}
 				}
 
+				//TODO: makevalid issues for 4326
 				// check if we need to clip and if we do build the clip region (tile extent)
 				var clipRegion *geom.Extent
```
Comment (meilinger) [inline atlas/map.go]: I had to comment out these lines in order to get rendering looking appropriate (L209-231)

**Comment by danielklim** [comment]
FYI, in order to get this branch to compile properly, you need to use `https://github.com/ProminentEdge/geom/tree/add-tilegrid-4326` instead of the standard `go-spatial/geom` lib.

**Comment by meilinger** [comment]
@danielklim correct.  With the new changes coming in https://github.com/go-spatial/geom/pull/99, this will have to be adapted.

**Comment by ARolek** [comment]
@meilinger  can you re-open this PR and set the base to v0.12.x_geom? The PR was closed accidentally but I still want to get this one into an upcoming release. Note that the geom migration is happening in a parallel branch so it can be merged into mainline when it's ready.

**Comment by meilinger** [comment]
@ARolek done, thanks!

**Pull request closed without merging**